### PR TITLE
Add Storybook to frost-web

### LINKED ISSUE #85: Add Storybook to frost-web
Storybook is a development environment for UI components. It allows you to browse a component library, view the different states of each component, and interactively develop and test components.

### PR DESCRIPTION
**Pull request by jwicks31**
Resolves #85

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,6 +81,12 @@
   "devDependencies": {
     "@po.et/stylelint-rules": "1.1.0",
     "@po.et/tslint-rules": "1.2.0",
+    "@storybook/addon-actions": "^3.4.5",
```
Comment (wzalazar) [inline package.json]: Can you pin the version of the packages?

File: src/stories/index.stories.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { withInfo } from "@storybook/addon-info";
```
Comment (wzalazar) [inline src/stories/index.stories.js]: Can you remove the semicolons?

File: src/stories/index.stories.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+import { text, boolean } from '@storybook/addon-knobs/react'
+
+  const wInfoStyle = {
+    header: {
+      h1: {
+        marginRight: "20px",
```
Comment (wzalazar) [inline src/stories/index.stories.js]: is possible to use the single quote?

File: src/stories/index.stories.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        fontSize: "25px",
+        display: "inline"
```
Comment (wzalazar) [inline src/stories/index.stories.js]: comma tailing?

File: src/stories/index.stories.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```
Comment (lautarodragan) [inline src/stories/index.stories.js]: Why is React needed here?

**Comment by lautarodragan** [comment]
@jwicks31 what's the status of this? Are you working on getting webpack to exclude Story Books from the build?

**Comment by jwicks31** [comment]
Yes. I'm having a little trouble with it. Walter is helping me with the explorer-web build.

**Comment by wzalazar** [comment]
@lautarodragan @jwicks31 it's ready to check!

**Comment by jwicks31** [comment]
LGTM! Thank you for all of the help on this!

**Pull request merged**